### PR TITLE
doc: Clean doc pre-release

### PR DESCRIPTION
**Pull request by gmuloc**
# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have run pre-commit for code linting and typing (`pre-commit run`)
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes (`tox -e testenv`)

### CODE REVIEW

File: docs/advanced_usages/custom-tests.md [hunk 1]
```diff
@@ -71,9 +71,6 @@ Full AntaTest API documentation is available in the [API documentation section](
 
 ### Instance Attributes
 
-!!! info
-    You can access an instance attribute in your code using the `self` reference. E.g. you can access the test input values using `self.inputs`.
-
```
Comment (carl-baillargeon) [inline docs/advanced_usages/custom-tests.md:76]: Why are we removing this? Irrelevant?

Comment (mtache) [reply docs/advanced_usages/custom-tests.md:76]: Agreed, it should be still relevant.

Comment (gmuloc) [reply docs/advanced_usages/custom-tests.md:76]: because people using ANTA as a python library should not be learning about self. We can put it back :p

Comment (mtache) [reply docs/advanced_usages/custom-tests.md:76]: +1 @gmuloc

File: docs/api/tests.routing.bgp.md [hunk 2]
```diff
@@ -18,3 +18,4 @@ anta_title: ANTA catalog for BGP tests
       filters:
         - "!test"
         - "!render"
+        - "!_.*"
```
Comment (carl-baillargeon) [inline docs/api/tests.routing.bgp.md:21]: Not sure if that's related but Inputs are showing empty:
image

File: docs/api/tests.routing.isis.md [hunk 3]
```diff
@@ -18,3 +18,4 @@ anta_title: ANTA catalog for IS-IS tests
       filters:
         - "!test"
         - "!render"
+        - "!_.*"
```
Comment (carl-baillargeon) [inline docs/api/tests.routing.isis.md:21]: Same here:
image

File: docs/cli/get-inventory-information.md [hunk 4]
```diff
@@ -7,9 +7,55 @@ anta_title: Retrieving Inventory Information
   ~ that can be found in the LICENSE file.
   -->
 
-The ANTA CLI offers multiple entrypoints to access data from your local inventory.
+The ANTA CLI offers multiple commands to access data from your local inventory.
 
-## Inventory used of examples
+## List devices in inventory
+
+This command will list all devices available in the inventory. Using the `--tags` option, you can filter this list to only include devices with specific tags (Visit [this page](tag-management.md) to learn more about tags).  The `--connected` option allows to display only the devices where a connection has been established.
```
Comment (carl-baillargeon) [inline docs/cli/get-inventory-information.md:14]:
```suggestion
This command will list all devices available in the inventory. Using the `--tags` option, you can filter this list to only include devices with specific tags (visit [this page](tag-management.md) to learn more about tags).  The `--connected` option allows to display only the devices where a connection has been established.
```

File: docs/api/tests.routing.bgp.md (hunk 2 above)
Comment (mtache) [reply docs/api/tests.routing.bgp.md:21]: Works on my machine.
But this has no effect on the private members.

File: docs/cli/tag-management.md [hunk 5]
```diff
@@ -197,3 +197,72 @@ spine1 :: VerifyUptime :: SUCCESS
...
+  "DC2-LEAF1B",
+  "DC2-SPINE1",
+  "DC2-SPINE2",
+  "LEAF",
+  "SPINE"
+]
+```
+
+!!! note
+    ANTA automatically injects the device name as device tag.
```
Comment (carl-baillargeon) [inline docs/cli/tag-management.md:268]: Not sure if it's necessary. It is mentionned above:
image

File: docs/advanced_usages/custom-tests.md (hunk 1 above)
Comment (carl-baillargeon) [reply docs/advanced_usages/custom-tests.md:76]: Haha ok good point :100:

File: docs/api/tests.routing.bgp.md (hunk 2 above)
Comment (mtache) [reply docs/api/tests.routing.bgp.md:21]: Forget my last comment, I was using mike instead of mkdocs :)

Comment (mtache) [reply docs/api/tests.routing.bgp.md:21]: It is related, I fixed it

File: anta/reporter/__init__.py [hunk 6]
```diff
@@ -141,7 +141,8 @@ def report_summary_tests(
     ) -> Table:
         """Create a table report with result aggregated per test.
 
-        Create table with full output: Test | Number of success | Number of failure | Number of error | List of nodes in error or failure
+        Create table with full output:
+        Test Name | # of success | # of skipped | # of failure | # of errors | List of failed or error nodes |
```
Comment (carl-baillargeon) [inline anta/reporter/__init__.py:145]:
```suggestion
        Test Name | # of success | # of skipped | # of failure | # of errors | List of failed or error nodes
```

**Review by carl-baillargeon — APPROVED** [review]
LGTM

File: docs/cli/tag-management.md [hunk 7]
```diff
@@ -197,3 +197,60 @@ spine1 :: VerifyUptime :: SUCCESS
...
+  "leaf",
+  "leaf1",
+  "leaf2",
+  "spine",
+  "spine1"
+]
+```
+
+!!! note
+    ANTA automatically injects the device name as device tag.
```
Comment (gmuloc) [inline docs/cli/tag-management.md:256]:
```suggestion
```

**Pull request merged**